.github/workflows/workflow_config.toml

```diff
@@ -25,13 +25,34 @@ closed = "Done"
 type = "pagure"
 enabled = true  # Enable/Disable Pagure instance
 instance_url = "https://pagure.io/"  # URL for pagure endpoint
+label = "cpe"  # Label marking issues to by synchronized
 blocked_label = "blocked"  # Label that is marking blocked issues
 usermap = "pagureio_jira_usermap.toml"  # Map pagure.io to JIRA users
 
+[[instances."pagure.io".query_repositories]]
+namespace = "fedora-infra"
+
 [instances."pagure.io".repositories]
 # List of repositories to check for tickets
 # When label is set, only tickets with this label will be covered
 "fedora-infrastructure" = {}
 "releng" = {}
 "centos-infra" = {}
-"fedora-infra/toddlers" = {label = "cpe"}
+"fedora-infra/ansible" = {enabled = false}  # The Ansible repo doesn’t have issues
```

Comment:
I never had `fedora-infra/ansible` in the configuration.

Reply:
This is to disable querying the `fedora-infra/ansible` repository, it is configured to be without issues.